### ISSUE TITLE
"help Invoke-Build -full" not showing help?

### ISSUE THREAD

**Issue by JVimes** [nightroman/Invoke-Build#76]
This has to be user error, but I installed the module per the readme, and now the help command just lists some metadata:

```powershell
PS> Install-Module InvokeBuild

Untrusted repository
You are installing the modules from an untrusted repository. If you trust this repository, change its InstallationPolicy value by running the Set-PSRepository cmdlet. Are you
sure you want to install the modules from 'PSGallery'?
[Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "N"): a

PS> Import-Module InvokeBuild
PS> help Invoke-Build -full

Name                              Category  Module                    Synopsis
----                              --------  ------                    --------
Invoke-Build                      Alias                               Invoke-Build.ps1
Invoke-Builds                     Alias                               Invoke-Builds.ps1

PS>
```

Am I supposed to see something else?

**Comment by nightroman** [comment]
This is the known issue, see [Get-Help Invoke-Build -Full doesn't work #41](https://github.com/nightroman/Invoke-Build/issues/41) and the [comment](https://github.com/nightroman/Invoke-Build/issues/41#issuecomment-267826756) with the remedy.

In short, as README says, use the full path to *Invoke-Build.ps1*. If you have ideas on how to improve the situation, please suggest. Otherwise, we can close the issue. There is not much I can do. Sorry for this annoyance.

**Comment by JVimes** [comment]
Ah, thanks! I wondered if that was the same issue. Took me a minute to find
where the scripts were installed. I'll take another look regarding a fix,
but feel free to close this issue.

On Jun 27, 2017 6:16 PM, "Roman Kuzmin" <notifications@github.com> wrote:

This is the known issue, see Get-Help Invoke-Build -Full doesn't work #41
<https://github.com/nightroman/Invoke-Build/issues/41> and the comment
<https://github.com/nightroman/Invoke-Build/issues/41#issuecomment-267826756>
with the remedy.

In short, as README says, use the full path to *Invoke-Build.ps1*. If you
have ideas on how to improve the situation, please suggest. Otherwise, we
can close the issue. There is not much I can do. Sorry for this annoyance.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/nightroman/Invoke-Build/issues/76#issuecomment-311521243>,
or mute the thread

**Comment by nightroman** [comment]
> Took me a minute to find where the scripts were installed.

```powershell
    (Get-Alias Invoke-Build).Definition
```

**Comment by nightroman** [comment]
I will add this tip to README.

**Comment by JVimes** [comment]
Perfect, that'll help. Thanks much!
